### PR TITLE
Graduating the Hand Coach Feature

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Simplified and cleaned up the Hand Coach properties and added tests to move the Hand Coach out of experimental

## Changes
- Fixes: #8996

### CODE REVIEW

**Comment by RogPodge** [comment]
To see the relevant property changes made look at the commits before the mass migration of files out of experimental.

**Review by cre8ivepark — APPROVED** [review]
Looks great! Could you please disable (hide) 'Experimental' badge in the Description Panel?
2021-01-11 11_16_55-Unity 2018 4 28f1 -  PREVIEW PACKAGES IN USE  - HandCoachExample unity - MRTK -

File: Assets/MRTK/SDK/Features/UX/Scripts/HandCoach/HandInteractionHint.cs [hunk 1]
```diff
@@ -33,60 +33,41 @@ public bool HideIfHandTracked
             }
         }
 
-        [Tooltip("Min delay for showing the visuals.  If the user's hands are not in view, the visuals will appear after min seconds.")]
+        [Tooltip("When the user's hands are not in view, the visuals will appear after HintDisplayDelay seconds.")]
         [SerializeField]
-        private float minDelay = 5f;
+        private float hintDisplayDelay = 5f;
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/HandCoach/HandInteractionHint.cs:38]: Can this have `PreviouslySerializedAs`?

File: Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset [hunk 2]
```diff
@@ -33,8 +33,8 @@ MonoBehaviour:
   contentScenes:
   - Name: MRTKExamplesHubMainMenu
     Path: Assets/MRTK/Examples/Experimental/ExamplesHub/Scenes/MRTKExamplesHubMainMenu.unity
-    Included: 0
-    BuildIndex: -1
+    Included: 1
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset:36]: We should revert this file. We typically leave it checked in with only the HandInteractionExamples scene included

File: Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab [hunk 3]
```diff
@@ -16110,17 +17174,19 @@ MeshRenderer:
   m_MotionVectors: 1
   m_LightProbeUsage: 1
   m_ReflectionProbeUsage: 1
+  m_RayTracingMode: 2
   m_RenderingLayerMask: 4294967295
   m_RendererPriority: 0
   m_Materials:
-  - {fileID: 2100000, guid: fa419ab56051229449e3b813df8f295f, type: 2}
+  - {fileID: 0}
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab:17181]: Does this material need to be re-added? Or is it okay without?

Comment (keveleigh) [reply Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab:17181]: There are a couple of these throughout the file

File: Assets/MRTK/SDK/Features/UX/Scripts/HandCoach/HandInteractionHint.cs [hunk 4]
```diff
@@ -33,60 +33,41 @@ public bool HideIfHandTracked
...
-                minDelay = value;
+                hintDisplayDelay = value;
             }
         }
 
-        [Tooltip("Max delay for showing the visuals.  If the user's hands are in view, the min timer will reset to 0, but the visuals will appear after max seconds.")]
+        [Tooltip("When the user's hands are in view, the visuals will appear after TrackedHandHintDisplayDelay seconds.")]
         [SerializeField]
-        private float maxDelay = 10f;
+        private float trackedHandHintDisplayDelay = 10f;
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/HandCoach/HandInteractionHint.cs:57]: Can this have `PreviouslySerializedAs`?

File: Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab (hunk 3 above)
Comment (RogPodge) [reply Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab:17181]: I'll go through this again (this was fixing an issue with the button not disappearing when it went beyond the clipping bounds)

**Pull request merged**